src/View/Http/DefaultRenderingStrategy.php

```diff
@@ -108,9 +108,11 @@ public function render(MvcEvent $e)
 
             $application = $e->getApplication();
             $events      = $application->getEventManager();
-            $e->setError(Application::ERROR_EXCEPTION)
-              ->setParam('exception', $ex);
-            $events->trigger(MvcEvent::EVENT_RENDER_ERROR, $e);
+
+            $e->setError(Application::ERROR_EXCEPTION);
+            $e->setParam('exception', $ex);
```

Comment:
Out of scope for this refactor. I am literally trying to do the bare
minimum to get the component working with the new EM version. Open a
separate PR for such changes, if you feel they're important.
On Oct 1, 2015 6:40 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In src/View/Http/DefaultRenderingStrategy.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40903929>:
>
> > @@ -108,9 +108,11 @@ public function render(MvcEvent $e)
> >
> >              $application = $e->getApplication();
> >              $events      = $application->getEventManager();
> > -            $e->setError(Application::ERROR_EXCEPTION)
> > -              ->setParam('exception', $ex);
> > -            $events->trigger(MvcEvent::EVENT_RENDER_ERROR, $e);
> > +
> > +            $e->setError(Application::ERROR_EXCEPTION);
> > +            $e->setParam('exception', $ex);
>
> Same as above: rely on a constant instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40903929>.
>